Code/Source/svFSI/mat_models.cpp

```diff
@@ -765,7 +790,7 @@ void get_pk2cc_dev(const ComMod& com_mod, const CepMod& cep_mod, const dmnType&
       }
 
       double r1 = J2d*mat_ddot(C, Sb, nsd) / nd;
-      auto S = J2d*Sb - r1*Ci;
```

Comment:
Kudos for finding that! Did you observe any differences in the [rate of convergence](https://github.com/SimVascular/svFSIplus/issues/197#issuecomment-1962182146) using ustruct?

Reply:
No I haven't. This bug caused S for the ustruct HO model to always be zero, which obviously led to big issues when using the ustruct HO model. (we actually don't have a ustruct HO test yet). I think the ustruct rate of convergence issue must be due to another bug.